handlers/main.yml

```
@@ -8,3 +8,6 @@
 
 - name: restart rsyslog-haproxy
   service: name=rsyslog state=restarted
+
+- name: reload systemd
+  command: /bin/systemctl daemon-reload
```

Prefer built-in modules when available

```suggestion
- name: reload systemd
  systemd:
    daemon_reload: yes

```